main.py

```diff
@@ -50,30 +57,34 @@ def draw_bars(self, year, month):
         raw_data = self.file_reader.read_weather_file(file)
         weather_data = [WeatherData.from_csv_row(row) for row in raw_data]
 
-        WeatherStats.find_daily_extremes(weather_data)
+        for data in weather_data:
+            if data.max_temp is not None and data.min_temp is not None:
+                print(
+                    f"{data.date} {Fore.BLUE + '+' * data.min_temp + Fore.RESET + Fore.RED + '+' * data.max_temp + Fore.RESET} {data.min_temp}C - {data.max_temp}C"
+                )
 
 
 if __name__ == "__main__":
     directory = sys.argv[1]
     weather_man = WeatherMan(directory)
 
     i = 2
-    
+
     while i < len(sys.argv):
         option = sys.argv[i]
 
         if option == "-e":
-            year = sys.argv[i+1]
+            year = sys.argv[i + 1]
             weather_man.process_year(year)
-            i+=2
+            i += 2
         elif option == "-a":
-            year, month = sys.argv[i+1].split('/')
+            year, month = sys.argv[i + 1].split("/")
             weather_man.process_month(year, month)
-            i+=2
+            i += 2
         elif option == "-c":
-            year, month = sys.argv[i+1].split('/')
+            year, month = sys.argv[i + 1].split("/")
             weather_man.draw_bars(year, month)
-            i+=2
+            i += 2
```

Comment:
Instead of writing i += 2 in every if write it outside the conditional statments